android/KMAPro/kMAPro/src/main/java/com/tavultesoft/kmapro/MainActivity.java

```diff
@@ -262,13 +262,15 @@ protected void onResume() {
 
     if (loadingIntentUri != null) {
       String scheme = loadingIntentUri.getScheme().toLowerCase();
+      final String link = loadingIntentUri.toString();
       switch (scheme) {
         // content:// Android DownloadManager
         // file:// Chrome downloads and Filebrowsers
         case "content":
         case "file":
           requestPermissionIntentUri = loadingIntentUri;
           if (CheckPermissions.isPermissionOK(this)) {
+            KMLog.LogBreadcrumb(TAG, "Installing local KMP: " + loadingIntentUri, true);
```

Comment:
Do we want `link`?

Reply:
Oh, I did miss one.  That said, it'll automatically run `.toString()` on it, so it's "fine".

Reply:
For clarity, consistency, and avoidance of unnecessary repetition, I think we should fix it anyway :grin:!

```suggestion
            KMLog.LogBreadcrumb(TAG, "Installing local KMP: " + link, true);
```